include/stdshims.h

```diff
@@ -0,0 +1,19 @@
+//! Shims for std:: functions that aren't available in the current C++ versions
+//! we target.
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#if (__cplusplus >= 201402L)
+#warning "This file should now be removed! The functions it provides are part of the C++14 standard."
+#endif
```

Comment:
Why don't you write this like this?
```c++
#if (__cplusplus >= 201402L)
#warning "This file should now be removed! The functions it provides are part of the C++14 standard."
using std::make_unique;
#else
// Shim
#endif
```
or write the function in the `std` namespace?

Reply:
Pushed an update using the code you listed above. Thanks!